dedicated-examples/mock-camera-microphone-access/README.md

```diff
@@ -1 +1,29 @@
-# mock camera/microphone access
+# Mock Camera/Microphone Access
+
+This example shows how to mock data for microphone and camera. This example runs in **Chrome**.
+
+**Test Code**: [mock-camera-microphone-access.js](mock-camera-microphone-access.js)
+
+The test page requests a permission to record audio and video. Resulting streams are directed to corresponding `<audio>` and `<video>` elements on the page. When the `<button>` element registers a click, a snapshot of the video stream draws on the `<img>` element if video stream is present.
```

Comment:
The test page requests **permission** to record audio and video. **The** resulting streams are directed to **the** corresponding `<audio>` and `<video>` elements on **a** page (or _the page_ if you are talking about a specific page. However, this isn't clear from the context) . When the `<button>` element registers a click, a snapshot of the video stream **is drawn** on the `<img>` element (if **a** video stream is present).